libraries/Microsoft.Bot.Builder.AI.QnA/QnAMakerAction.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Dialogs.Adaptive;
+using Microsoft.Bot.Builder.Dialogs.Adaptive.Events;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// QnAMaker dialog which uses QnAMaker to get an answer.
+    /// </summary>
+    /// <remarks>
+    /// The answer is treated as a inline LG expression, so it can be completely customized by LG system.
+    /// </remarks>
+    public class QnAMakerAction : Dialog
```

Comment:
I'm not understanding that an Action is a dialog. Is this the right naming scheme?

Reply:
I feel like QnAMakerDialog is a better name. @tomlm and I discussed this and thought QnAMakerDialog would be preferred.

Reply:
As far as discussion is going on for this version and the way that we are placing QnAMaker in Composer, ie with other actions. So, we thought of keeping the name as QnAMakerAction